### PR TITLE
Azure CICD Infrastructure Rebuilds Failures

### PR DESCRIPTION
**Pull request by BRMcLaren**
Fixes 2 current issues in the CICD infrastructure.
    * Increase managed disk build time by double, fixes intermittent issues caused by RHEL8 upload due to rhel image size
    * Update VSCode Hash
    * Remove Node URL from windows ACC ansible
    * Removed unused package testing Jenkins file

### CODE REVIEW

**Comment by BRMcLaren** [comment]
bors try

**Comment by BRMcLaren** [comment]
bors try

File: scripts/ansible/oe-windows-acc-setup.yml [hunk 1]
```diff
@@ -17,9 +17,7 @@
         seven_zip_url:       "https://oejenkins.blob.core.windows.net/oejenkins/7z1806-x64.msi"
         seven_zip_hash:      "F00E1588ED54DDF633D8652EB89D0A8F95BD80CCCFC3EED362D81927BEC05AA5"
         vs_buildtools:       "https://oejenkins.blob.core.windows.net/oejenkins/vs_buildtools_2017.exe"
-        vs_buildtools_hash:  "6F49872B04A0EAEDF5ED96AB25F7697062D81A419D45D9970E41784F31165BF2"
-        node_url:            "https://oejenkins.blob.core.windows.net/oejenkins/node-v10.16.3-x64.msi"
-        node_hash:           "f68b75eea46232adb8fd38126c977dc244166d29e7c6cd2df930b460c38590a9"
+        vs_buildtools_hash:  "9822B4C851E14D9658BABD1533F66F518C6169196E985FE5713B2774128832AE"
```
Comment (anakrish) [inline scripts/ansible/oe-windows-acc-setup.yml:20]: How was this working before with incorrect hash?

File: .jenkins/infrastructure/build_azure_managed_images.Jenkinsfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ import java.time.format.DateTimeFormatter
 OECI_LIB_VERSION = env.OECI_LIB_VERSION ?: "master"
 oe = library("OpenEnclaveCommon@${OECI_LIB_VERSION}").jenkins.common.Openenclave.new()
 
-GLOBAL_TIMEOUT_MINUTES = 240
+GLOBAL_TIMEOUT_MINUTES = 480
```
Comment (anakrish) [inline .jenkins/infrastructure/build_azure_managed_images.Jenkinsfile:10]: Why is 480 a better number? Isn't 8 hours a really long time?

Comment (BRMcLaren) [reply .jenkins/infrastructure/build_azure_managed_images.Jenkinsfile:10]: Yup, it is a really long time. But we can cancel the job if need be, this is the Jenkins job that rebuilds all the infrastructure and it takes a long time. Rhel, for example, takes forever: https://oe-jenkins-tf.westeurope.cloudapp.azure.com/job/CI-CD_Infrastructure/job/OpenEnclave-Rebuild-Infrastructure/

**Comment by BRMcLaren** [comment]
bors try

File: scripts/ansible/oe-windows-acc-setup.yml (hunk 1 above)
Comment (CodeMonkeyLeet) [reply scripts/ansible/oe-windows-acc-setup.yml:20]: If the cached version of this package is also updated on the backend, have we considered, caching multiple versions that can be referenced via different URLs (e.g. `https://oejenkins.blob.core/windows.net/oejenkins/v3/vs_buildtools_2017.exe`)? It seems brittle today that the update and merge of the oe-windows-acc-setup.yml needs to be synchronized with updates of the cached copies of the targets.

**Pull request merged**